idd/Energy+.idd.in

```diff
@@ -24564,9 +24564,9 @@ ZoneEarthtube:Parameters,
       \units dimensionless
       \minimum 0.25
       \maximum 1.0
-      \default 1.0 
- N4,  \field Earth Tube Dimensionless Boundary Above
-      \note When set to 1.0, the upper boundary is one earth tube radius below ground.
+      \default 1.0
+ N4,  \field Earth Tube Dimensionless Boundary Below
+      \note When set to 1.0, the below boundary is one earth tube radius below the earth tube.
```

Comment:
@RKStrand  Could you suggest the correct \note here please (I know what I wrote is wrong)
cf https://github.com/NREL/EnergyPlus/pull/10138#pullrequestreview-1622406516

Reply:
Suggested correction: When set to 1.0, the depth of the solution space is equal to the maximum vertical dimension above the earth tube.  This maximum dimension above the earth tube is the depth of the earth tube minus three times the diameter of the earth tube itself.  This sets the depth of the solution space below the earth tube node.

Reply:
But in N3, the note says the maximum dimension above the earth tube is the maximum vertical dimension minus 1 earth tube radius? Should this note say "...minus two times the radius...". This stood out because it is so different from the note above. These 2 notes should look very similar?

    \note When set to 1.0, the upper boundary is one earth tube radius below ground.

Reply:
Yes, @rraustad, I was trying to write a quick response before having to head off to classes.  I probably should have waited.  Here is another attempt to clarify both of these notes:

For N3: When set to 1.0, the total thickness of the solution space above the earth tube node is equal to the maximum vertical dimension above the earth tube. This maximum dimension above the earth tube is the depth of the earth tube minus three times the radius of the earth tube itself. This sets the depth of the solution space above the earth tube node only.  The earth tube node thickness is four times the earth tube radius so when this parameter is set to 1.0, the top node of the solution space is one earth tube radius below the ground surface.  When this parameter is less than 1.0, the solution space thickness above the earth tube is simply this parameter times the maximum thickness defined by the earth tube depth and the earth tube radius.

For N4: When set to 1.0, the total thickness of the solution space below the earth tube node is equal to the maximum vertical dimension above the earth tube. This maximum dimension above the earth tube is the depth of the earth tube minus three times the radius of the earth tube itself. This sets the depth of the solution space below the earth tube node only.  The earth tube node thickness is four times the earth tube radius so when this parameter is set to 1.0, the bottom node of the solution space is twice the earth tube depth minus the earth tube radius below the ground surface.  When this parameter is less than 1.0, the solution space thickness below the earth tube is simply this parameter times the maximum thickness defined by the earth tube depth and the earth tube radius.

Hopefully this makes sense now?  Sorry for all the confusion!

Reply:
Yes, but I'm not sure what to put there. The IDD shouldn't replace the IO Ref guide and this seems like a ton of text.
Maybe just keep the first line? I'll set suggestions